inc/dps/json.h

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file
+ * JSON <-> CBOR conversion
+ */
+
+/*
+ *******************************************************************
+ *
+ * Copyright 2016 Intel Corporation All rights reserved.
+ *
+ *-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+ */
+
+#ifndef _DPS_JSON_H
+#define _DPS_JSON_H
+
+#include <stdint.h>
+#include <dps/err.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup json JSON
+ * Convert between JSON and CBOR
+ * @{
+ */
+
+/**
+ * Generate CBOR from a JSON string
+ *
+ * @param json     NUL terminated JSON string to convert
+ * @param cbor     Destination buffer for the conversion
+ * @param cborSize The size of the cbor buffer
+ * @param cborLen  Returns the number of CBOR bytes written 
+ *
+ * @return - DPS_OK if the conversion was successful
```

Comment:
Doxygen doesn't format the bullets correctly unless you do:
@return 
- bullet 1
- bullet 2

Reply:
Will fix